src/main/java/com/radixdlt/crypto/hdwallet/BitcoinJMnemonicToSeedConverter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *
+ *  * (C) Copyright 2020 Radix DLT Ltd
+ *  *
+ *  * Radix DLT Ltd licenses this file to you under the Apache License,
+ *  * Version 2.0 (the "License"); you may not use this file except in
+ *  * compliance with the License.  You may obtain a copy of the
+ *  * License at
+ *  *
+ *  *  http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing,
+ *  * software distributed under the License is distributed on an
+ *  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *  * either express or implied.  See the License for the specific
+ *  * language governing permissions and limitations under the License.
+ *
+ */
+
+package com.radixdlt.crypto.hdwallet;
+
+
+import org.bitcoinj.crypto.MnemonicCode;
+
+import java.util.List;
+import java.util.Objects;
+
+import static org.bitcoinj.core.Utils.WHITESPACE_SPLITTER;
+
+final class BitcoinJMnemonicToSeedConverter {
+
+	private BitcoinJMnemonicToSeedConverter() {
+		throw new IllegalStateException("Can't construct.");
+	}
+
+	public static byte[] seedFromMnemonicAndPassphrase(List<String> words, String passphrase) throws MnemonicException {
```

Comment:
Static methods should not be public in a package-local class.  Because the class is package-local, the public modifier is effectively hidden here.